.github/workflows/build.yml

```diff
@@ -26,7 +27,7 @@ jobs:
       if: steps.get-docker-hub-username.outputs.dockerhub_username
     - name: Set MARKET_URL var for external users
       run: |
-          echo "MARKET_URL=https://staging.dev.medicmobile.org" >> $GITHUB_ENV
+          echo "AUTH_MARKET_URL=https://staging.dev.medicmobile.org" >> $GITHUB_ENV
           echo "BUILDS_SERVER=_couch/builds_external" >> $GITHUB_ENV
           echo "STAGING_SERVER=_couch/builds_external" >> $GITHUB_ENV
       if:  ${{ !env.MARKET_URL }}
```

Comment:
I think this line will need to change too because MARKET_URL will always be set so external contributors will not work. This is repeated in multiple places through the code. It may be worth looking in to an alternative so this can be set in one place and not copied and pasted everywhere, for example, can we use some form of ternary operation when setting environment variables (line 10), eg:
`STAGING_SERVER: ${{ secrets.AUTH_MARKET_URL ? "_couch/builds" : "_couch/builds_external" }}`

Reply:
I am working on cleaning up the `build.yml` as part of another change. Hopefully reducing it down to only three tasks and not a bunch of repeated code for tests. 

A quick search it seems GH actions don't support ternaries in expression yet.